### PR TITLE
OOP school library: unit tests

### PR DESCRIPTION
**Pull request by Yerimah**
In this branch, we added unit tests to all the classes using Rspec

### CODE REVIEW

File: spec/book_spec.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    it 'checks if the title of book is Mindset' do
+      expect(@book.title).to eq('Mindset')
+    end
+
+    it 'checks if author of book is Carol Dweck' do
+      expect(@book.author).to eq('Carol Dweck')
+    end
+  end
+end
```
Comment (julie-ify) [inline spec/book_spec.rb]: - [ ] Kindly add a test for the `add_rentals` method in the Book class

File: spec/person_spec.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+require_relative '../person'
+
+describe Person do
+  context 'Test for the person class' do
```
Comment (julie-ify) [inline spec/person_spec.rb:4]: - [ ] Kindly add a test for the `add_rentals` method in the Person class

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @Yerimah ane @BenjaminSemah,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Tests are well written and passed ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kingstalux — APPROVED** [review]
Hi @Yerimah and @BenjaminSemah,
# Status: Approved ✔️ ✔️ ✔️  

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966

## Highlights ⚡ 
✅ All tests are passing 🔥 
✅ Good pull request description 👍 
✅ Professional README file 💯 
✅ Good commit messages 👏 

Good job implementing the required changes 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**